integration/update_array_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+// This file is for tests of array update operators:
+// - `$`
+// - `$[]`
+// - `$[<identifier>]`
+// - `$addToSet`
+// - `$pop`
+// - `$pull`
+// - `$push`
+// - `$pullAll`.
+//
+// And array update operator modifiers:
```

Comment:
There is a single group, not two groups

![image](https://user-images.githubusercontent.com/11512/176684525-e4a84101-769d-47c6-aeab-8c41a8c359c5.png)

Reply:
> There is a single group, not two groups
> 
> ![image](https://user-images.githubusercontent.com/11512/176684525-e4a84101-769d-47c6-aeab-8c41a8c359c5.png)

done.